charts/posthog/templates/events-deployment.yaml

```diff
@@ -101,6 +101,12 @@ spec:
         # statsd env variables
         {{- include "snippet.statsd-env" . | indent 8 }}
 
+        # async migrations will be run by the migrate job
+        {{- if .Values.migrate.enabled -}}
+        - name: SKIP_ASYNC_MIGRATIONS_SETUP
+          value: 'True'
+        {{- end }}
```

Comment:
might cause some problems if self-hosted users try to bypass too far to the future, but that's low priority, so 🚢  🚀